src/Sylius/Bundle/CoreBundle/Migrations/Version20220803125615.php

```diff
@@ -14,7 +14,7 @@
 namespace Sylius\Bundle\CoreBundle\Migrations;
 
 use Doctrine\DBAL\Schema\Schema;
-use Doctrine\Migrations\AbstractMigration;
+use Sylius\Bundle\CoreBundle\Doctrine\Migrations\AbstractMigration;
 
 /**
```

Comment:
We'll create a PR removing all these comments, as some migrations have it, and some don't.